stylesheets/components/image.css

```diff
@@ -0,0 +1,15 @@
+.img {
+  height: 150px;
+  width: 150px;
+  object-fit: cover;

```

Comment:
You can shift this code to reset or img tag.

Reply:
`img` class will be specifically used with default height and width that's why can't give it in reset css.